### ISSUE TITLE
Add required function to API REST templates in Helm Chart

### ISSUE THREAD

**Issue by maruiz93** [kubearchive/kubearchive#91]
## Required function in Helm Templates
The optional fields in the kubernetes resources that we expect to be filled through the `helm template` command should include this function to detect typos or similar errors in the CI.

More information on the function in [here](https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-required-function)

## Goal
Include this keyword in the templates of the kubearchive REST API component.

**Comment by rh-hemartin** [comment]
Seeing the implementation on #211, it does not convince me to have a bunch of "required" scattered all over the place. What about using a schema? https://helm.sh/docs/topics/charts/#schema-files

**Comment by maruiz93** [comment]
I have nothing against the schema but, I think it does not cover the same functionality.

With the `required` function I am preventing that if I make a typo on the name of the variable either in the `values.yaml` or in a template where I reference a variable name, it will fail on CI. 

With the schema, we make sure that several values for variables are provided, that covers having typos or lacking a value for a variable in the `values.yaml`, but what if I make a typo referencing the variable in the template? It will be rendered as an empty value.

The `required` function is the better I found for covering that, but maybe there are alternatives. For me, the best would be an option in the `helm lint` command that ensures that every variable referenced in a template has to be defined, but  the more similar thing I found for that is the `required` function.
